cmake/showoptions.cmake

```diff
@@ -134,6 +134,11 @@ if(WITHOUT_METRICS)
   message(" *** WITHOUT_METRICS - WARNING!")
   message(" *** Please note that this will disable all metrics output (i.e. InfluxDB and Grafana)")
   add_definitions(-DWITHOUT_METRICS)
+elseif (WITH_DETAILED_METRICS)
+  message("")
+  message(" *** WITH_DETAILED_METRICS - WARNING!")
+  message(" *** Please note that this will enable detailed metrics output (i.e. time each session takes to update)")
+  add_definitions(-DWITHOUT_METRICS)
```

Comment:
```suggestion
  message(" *** Please note that this will enable detailed metrics output (i.e. time each session takes to update)")
  add_definitions(-DWITH_DETAILED_METRICS)
```